app/assets/javascripts/pretty-text/addon/emoji.js

```diff
@@ -180,6 +180,52 @@ export function buildEmojiUrl(code, opts) {
   return url;
 }
 
+export function countEmoji(string, opts) {
+  const allTranslations = Object.assign(
+    {},
+    translations,
+    opts.customEmojiTranslation || {}
+  );
```

Comment:
I'm curious, does the following work in miniracer?

```suggestion
  const allTranslations = {
    ...translations,
    ...(opts.customEmojiTranslation || {})
  };
```